src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -45,6 +44,13 @@ public static void GetDirectoryName(string path, string expected)
             Assert.Equal(expected, Path.GetDirectoryName(path));
         }
 
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, ".Net Core returns null dotnet/corefx #19337")]
+        public static void GetDirectoryName_EmptyString_Throws()
+        {
+            Assert.Throws<ArgumentException>(() => Path.GetDirectoryName(string.Empty));
+        }
```

Comment:
I opened issues for that and fixed this temporary as the .Net Core fix has to go on coreclr, and then wait for coreclr to be ingested, so I was planning on fix the tests when this is fixed on coreclr and ingested.